Sources/IMADAIPlugin.swift

```diff
@@ -196,7 +196,10 @@ import PlayKitUtils
             throw IMADAIPluginRequestError.missingPlayerView
         }
         
-        adDisplayContainer = IMADAIPlugin.createAdDisplayContainer(forView: playerView, viewController: playerView.findViewController(), withCompanionView: pluginConfig.companionView)
+        let displayContainer = IMADAIPlugin.createAdDisplayContainer(forView: playerView,
+                                                                     viewController: playerView.findViewController(),
+                                                                     withCompanionView: pluginConfig.companionView)
+        adDisplayContainer = displayContainer
```

Comment:
@chausov  why not to use it as it is today?

Reply:
adDisplayContainer defined as optional but in IMALiveStreamRequest it required non optional adDisplayContainer: as a parameter.
we may change the definition of adDisplayContainer to be non optional property it will be better.

This changes required because in IMA SDK some of the types was changed